_monitoring-your-cluster/pa/rca/shard-hotspot.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Shard hotspot identification
+parent: Root Cause Analysis
+grand_parent: Performance Analyzer
+nav_order: 30
+---
+
+## Shard hotspot identification
+
+With the shard hotspot identification RCA, you can identify a hot shard within an index. A hot shard is an outlier that consumes more resources than its counterparts. The shard hotspot identification RCA monitors the following metrics:
+
+- `CPU_Utilization`
+- `Heap_AllocRate`
+
+These metrics provide an accurate picture of operation intensities for certain shards, such as the following: 
+
+- Bulk requests - High heap allocation rate.
+- Search requests - High CPU utilization, more complex queries, and heap allocation rates.
+- Document updates - Relatively high balance between `CPU_Utilization` and `Heap_AllocRate`.
+
+The shard hotspot identification RCA looks at the `CPU_Utilization` and `Heap_AllocRate` metric data and compares the values against the threshold for each resource. If the usage for any of these resources is greater than their individual threshold, the context is marked as "unhealthy" and creates a "Hot Shard Resource Summary" for the shard.
+
+For in-depth information regarding the operation of the shard hotspot identification RCA, see the following [Github readme](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/hotshard/docs/README.md).
+
+#### Example request
+
+```bash
+GET localhost:9600/_plugins/_performanceanalyzer/rca?name=HotShardClusterRca
```

Comment:
Nit, should we remove host/port from the example? If someone were running from a remote cluster, they would need to modify the example.

Reply:
I have this updated